goon/code/datums/browserOutput.dm

```diff
@@ -175,7 +175,7 @@ For the main html chat area
 			//Uh oh this fucker has a history of playing on a banned account!!
 			if (found.len > 0)
 				//TODO: add a new evasion ban for the CURRENT client details, using the matched row details
-				message_admins("<FONT SIZE=+2 COLOR=red>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</font>")
+				message_admins("<FONT SIZE=+2 span class='danger'>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</font>")
```

Comment:
```
+				message_admins("<FONT SIZE=+2 span class='danger'>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</font>")
```
```
+				message_admins("<span class='danger'>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</span>")
```

Reply:
I did test it before sending the commit and it did properly display it. But I'll change it up again tomorrow.

Reply:
This is the result after changing it and it isn't as large as I wanted it to be. The reason I used font size was to make it larger than the regular text.

![dreamseeker_2017-08-27_17-39-57](https://user-images.githubusercontent.com/19466872/29751521-d8b7d11c-8b4e-11e7-9951-ae7be3a4df3c.png)